api/models/workshop.py

```diff
@@ -23,6 +25,10 @@ class Workshop(CapacityMixin, Base):
         WorkshopDifficulty,
         verbose_name=_("Difficulty"),
     )
+    location = models.ForeignKey(
```

Comment:
@PetrDlouhy, so you propose we have same workshop locations for every year? Now that's insane.